src/main/java/org/cyclonedx/gradle/CycloneDxTask.java

```diff
@@ -76,6 +77,7 @@
     private boolean skip;
     private final List<String> skipConfigs = new ArrayList<>();
 
+    @Input
```

Comment:
The gradle plugin helper did not like that this field did not declare it was an input for the plugin.  There are other fields that could be also specified as inputs (i.e. buildDir), but they need getters to be treated as such.